packages/foam-core/test/utils.test.ts

```diff
@@ -53,7 +62,7 @@ describe('hashtag extraction', () => {
 
   it('ignores hashes in links to sections', () => {
     expect(
-      extractHashtags(`
+      extractHashtags(`first line not blank
```

Comment:
why does the first line need not be blank?

Reply:
The basic Markdown syntax allows to create code blocks by indenting lines by more than four spaces or one tab. So an indented block after a blank line would be considered as a block of code.

Links could be found https://spec.commonmark.org/0.30/#indented-code-block  and
https://www.markdownguide.org/basic-syntax#code-blocks.

Reply:
This is a really good point. 
Purely to make life easier for people reading the test in the future (who will probably wonder the same thing, and won't have access to your answer), I wonder if we should instead keep the first line blank, but remove the indentation from the following lines (except in the tests cases that will check the behavior in code blocks).
what do you think?

Reply:
I think it would be more concise to remove the indentation rather than to add the first line.

Reply:
sounds good, let's go with that!